src/main/java/com/commercetools/paymenttoorderprocessor/jobs/actions/MessageReader.java

```diff
@@ -82,7 +81,7 @@ private void getUnprocessedMessagesFromPlatform() {
                 .filter(message -> messageProcessedManager.isMessageUnprocessed(message))
                 .collect(Collectors.toList());
 
-        LOG.info("{} of {} messages on current page are unprocessed", messages.size(), result.size());
+        LOG.info("total {} messages, {} on current page, {} of them are are unprocessed", total, result.size(), messages.size());
```

Comment:
i'd say this log exactly fits "info" definition: it is an information about the application execution and show how the app going on. If something is crashed in the future we at least have some checkpoints of the application execution.